src/docs/organization.yaml

```diff
@@ -218,7 +218,49 @@ paths:
                                     message:
                                         type: string
                                         example: 'Invalid Token: No authorization token was found'
-
+    /api/organization/{id}/sub:
+        post:
+            tags:
+                - Organization
+            summary: Delete organization sub
+            description: Delete organization sub
+            operationId: deleteSubscription
+            consumes:
+                - application/json
+            produces:
+                - application/json
+            parameters:
+                - in: path
+                  name: id
+                  description: The id of the organization
+                  required: true
+                  type: string
+            responses:
```

Comment:
Can you please add thıs response code, since your endpoint need an authenticated user: 
```
403:
  description: This status is returned when the user is not authorized to perform the operation.
```